pkg/kubectl/kubectl.go

```diff
@@ -51,6 +51,10 @@ var ResourcesShortcutStatic = []ResourceShortcuts{
 		LongForm:  schema.GroupResource{Resource: "configmaps"},
 	},
 	{
+		ShortForm: schema.GroupResource{Resource: "crd"},
+		LongForm:  schema.GroupResource{Resource: "customresourcedefinition"},
```

Comment:
I'm afraid that we shouldn't add this here, because seems like it had been added somewhere else(I didn't find where, though. Maybe https://github.com/kubernetes/apiextensions-apiserver). Running `kubectl get crd` on HEAD already works. That why I didn't add it in #49088:)